test/ejabberd_tests/tests/rest_SUITE.erl

```diff
@@ -183,6 +185,53 @@ messages_are_archived(Config) ->
         BobJID = maps:get(sender, Previous)
     end).
 
+send_and_wait(AliceJID, BobJID, Msg) ->
+    M = #{caller => BobJID, to => AliceJID, msg => Msg},
+    {?OK, _} = post(<<"/messages">>, M),
+    M1 = #{caller => AliceJID, to => BobJID, msg => Msg},
+    {?OK, _} = post(<<"/messages">>, M1),
+    timer:sleep(2000).
```

Comment:
It is not about checking - to test search I need to build an archive, so I'm sending messages every two seconds. When after sending the last one I go back three seconds I can be pretty sure I'm going to skip the last two messages.

Timestamp is is in seconds because it is eventually going to be used by web clients, I think it is more natural for them to use seconds.